src/process-dir.js

```diff
@@ -28,28 +26,36 @@ export const processDir = async (rootPath, excludedPaths = []) => {
     isFolder = true,
   ) => {
     try {
-      console.log("Looking in ", path);
+      console.log("Looking in ", `./${path}`);
+
       if (isFolder) {
-        const files = await fs.readdirSync(path);
-        const children = []
-        for (const file of files) {
-          if (fullPathFoldersToIgnore.includes(rootPath + "/" + file)) {
+        const filesOrFolders = await fs.readdirSync(`./${path}`);
+        const children = [];
+
+        for (const fileOrFolder of filesOrFolders) {
+          const fullPath = nodePath.join(rootPath, fileOrFolder);
```

Comment:
I think `rootPath` needs to be `path`, to include the parents recursively?

Reply:
You're absolutely right, that was a bug :)